modules/home-manager/hyprlock/conf.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/hyprlock.conf b/hyprlock2.conf
+index f976b90..a6f2867 100644
+--- a/hyprlock.conf
++++ b/hyprlock2.conf
+@@ -1,7 +1,3 @@
+-source = $HOME/.config/hypr/mocha.conf
+-
+-$accent = $mauve
+-$accentAlpha = $mauveAlpha
+ $font = JetBrainsMono Nerd Font
+ 
+ # GENERAL
+@@ -13,7 +9,6 @@ general {
+ # BACKGROUND
+ background {
+     monitor =
+-    path = ~/.config/background
+     blur_passes = 0
+     color = $base
+ }
+@@ -42,19 +37,6 @@ label {
+     valign = top
+ }
+ 
+-# USER AVATAR
+-
+-image {
```

Comment:
As far as I understand, there is no consistent way to get user's avatar, so I removed it to make it work from the box